test/functional/test_framework/util.py

```diff
@@ -67,6 +72,16 @@ def assert_greater_than_or_equal(thing1, thing2):
         raise AssertionError("%s < %s" % (str(thing1), str(thing2)))
 
 
+def assert_less_than(thing1, thing2):
+    if thing1 >= thing2:
+        raise AssertionError("%s .= %s" % (str(thing1), str(thing2)))
```

Comment:
2103e12: here and for `assert_less_than_or_equal` too.
```suggestion
        raise AssertionError("%s >= %s" % (str(thing1), str(thing2)))
```